staging/src/k8s.io/kubectl/pkg/cmd/set/set_resources.go

```diff
@@ -267,7 +267,7 @@ func (o *SetResourcesOptions) Run() error {
 
 		//no changes
 		if string(patch.Patch) == "{}" || len(patch.Patch) == 0 {
-			allErrs = append(allErrs, fmt.Errorf("info: %s was not changed\n", name))
+			fmt.Fprintln(o.ErrOut, fmt.Sprintf("set resources for %s cancelled, no changes detected", name))
```

Comment:
Drop this line entirely, just `continue` is sufficient. While at it please fix this also in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/set/set_subject.go

Reply:
ok, done